samples/nrf9160/modem_shell/README.rst

```diff
@@ -499,6 +499,29 @@ Examples
 
      rest -d example.com -l 1024 -m head -H "X-foo1: bar1\x0D\x0A" -H "X-foo2: bar2\x0D\x0A"
 
+----
+
+nRF Cloud
+=========
+
+MoSh command: ``cloud``
+
+nRF Cloud is a platform for providing, among other things, various location services. Modem shell
+enables the user to establish an MQTT connection to nRF Cloud by using :ref:`lib_nrf_cloud`.
+Currently, ``cloud`` command is useful mostly when utilizing the Location tool and MQTT is the
+desired transport protocol. However, there is no limit on what nRF Cloud services can be used once
+the MQTT connection is established.
```

Comment:
```suggestion
nRF Cloud is a platform for providing, among other things, various location services.
Modem shell enables you to establish an MQTT connection to nRF Cloud using the :ref:`lib_nrf_cloud` library.
Currently, the ``cloud`` command is useful mostly when using the location services and MQTT is the desired transport protocol.
However, you can use any nRF Cloud services once the MQTT connection is established.
```

Reply:
Thanks. I updated also the example...it was a bit too trivial.